include/XLink/XLinkPublicDefines.h

```diff
@@ -126,6 +127,8 @@ typedef struct streamPacketDesc_t
 {
     uint8_t* data;
     uint32_t length;
+    struct timespec trsend;
+    struct timespec treceive;
```

Comment:
I'd just augment the variable naming or add a `/// comment` describing a bit more:
```suggestion
    struct timespec tRemoteSent; /// remote timestamp of when the packet was sent. Related to remote clock. Note: not directly related to local clock
    struct timespec tReceived; /// local timestamp of when the packet was received. Related to monotonic clock
```